firstvoices/backend/serializers/base_serializers.py

```diff
@@ -89,12 +90,37 @@ def create(self, validated_data):
 
 class CreateControlledSiteContentSerializerMixin(CreateSiteContentSerializerMixin):
     """
-    A mixin for ModelSerializers that sets the required fields for subclasses of BaseSiteContentModel as well as the
-    visibility field
+    A mixin for ModelSerializers that sets the required fields for subclasses of BaseControlledModel
     """
 
-    def create(self, validated_data):
-        validated_data["visibility"] = Visibility[
-            str.upper(validated_data.pop("get_visibility_display"))
-        ]
-        return super().create(validated_data)
+    def validate(self, attrs):
```

Comment:
Should a public or member site be able to have team objects created by an assistant? Currently the site must also be team for the assistant to create